modular_skyrat/modules/assaultops/code/gamemode/assaultops.dm

```diff
@@ -0,0 +1,275 @@
+	gloves =  /obj/item/clothing/gloves/combat
+	back = /obj/item/storage/backpack/fireproof
+	ears = /obj/item/radio/headset/syndicate/alt
+	l_pocket = /obj/item/modular_computer/tablet/nukeops
+	id = /obj/item/card/id/syndicate/nuke_leader
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/aps
+	r_pocket = /obj/item/ammo_box/magazine/m9mm_aps
+	belt = /obj/item/storage/belt/utility/syndicate
+
+	var/command_radio = FALSE
+	var/cqc = FALSE
+
+/datum/outfit/assaultops/cqb
+	name = "Assault Operative - CQB"
+
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/c20r,\
+		/obj/item/ammo_box/magazine/smgm45=4,\
+		)
+
+	cqc = TRUE
+
+/datum/outfit/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+	belt = /obj/item/storage/belt/grenade/full
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/gyropistol,\
+		/obj/item/ammo_box/magazine/m75=4,\
+		/obj/item/implant/explosive/macro, \
+		/obj/item/storage/box/assaultops/demoman
+		)
+	l_hand = /obj/item/clothing/suit/space/hardsuit/rd
+
+/obj/item/storage/box/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+/obj/item/storage/box/assaultops/demoman/PopulateContents()
+	for(var/i in 1 to 4)
+		new /obj/item/grenade/syndieminibomb(src)
+		new /obj/item/grenade/c4(src)
+
+/datum/outfit/assaultops/medic
+	name = "Assault Operative - Medic"
+
+	glasses = /obj/item/clothing/glasses/hud/health
+	belt = /obj/item/storage/belt/medical/paramedic
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/submachine_gun/pps,\
+		/obj/item/ammo_box/magazine/pps=4,\
+		/obj/item/storage/firstaid/tactical=2,\
+		/obj/item/gun/medbeam)
+
+/datum/outfit/assaultops/heavy
+	name = "Assault Operative - Heavy Gunner"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/l6_saw/unrestricted/mg34/assaultops,\
+		/obj/item/ammo_box/magazine/mg34=4,\
+		/obj/item/grenade/syndieminibomb)
+
+/obj/item/gun/ballistic/automatic/l6_saw/unrestricted/mg34/assaultops
+	w_class = WEIGHT_CLASS_NORMAL
+
+/datum/outfit/assaultops/assault
+	name = "Assault Operative - Assault"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/assault_rifle/akm,\
+		/obj/item/ammo_box/magazine/akm=4,\
+		/obj/item/grenade/syndieminibomb=2)
+
+/datum/outfit/assaultops/sniper
+	name = "Assault Operative - Sniper"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
```

Comment:
They're an sniper, why would they wear a elite hardsuit?

Reply:
Hmm, something more fitting of a sniper role could be a hardsuit with reactive stealth armor properties?

Reply:
That could work, yeah